etc/yoda/app-loader-config.json

```diff
@@ -1,6 +1,7 @@
 {
   "paths": [
-    "/opt/apps"
+    "/opt/apps",
+    "/data/apps"
```

Comment:
Currently apps on `/data/apps` should not be enabled by default.

Reply:
Yea, should we append the `/data/apps` for developers by default?

Reply:
I'd suggest another issue/PR for behavior changes.